### PR TITLE
Owner check fix

### PR DESCRIPTION
**Pull request by wersal454**
## What type of PR is this?
1. [x] Bug
2. [ ] Change
3. [ ] Enhancement
4. [ ] Miscellaneous

### What have you changed and why?
Information:

Possible check fix for moveout crew button when players cannot move crew if vehicle is not player's or if owner is not defined

Not tested

### Please specify which Issue this PR Resolves (If Applicable).
"This PR closes #XXXX!"

### Please verify the following.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:

********************************************************
Notes:

### CODE REVIEW

**Comment by Silence-112** [comment]
Add more info on what this fixes pls
I know, but other people don't

File: A3A/addons/core/functions/Base/fn_moveOutCrew.sqf [hunk 1]
```diff
@@ -42,7 +42,7 @@ if (isNull _vehicle) exitWith {
     [localize "STR_A3A_Base_moveOutCrew_header", localize "STR_A3A_reinf_airstrike_not_looking_at_veh"] remoteExecCall ["SCRT_fnc_misc_deniedHint",_player];
 };
 _owner = _vehicle getVariable ["ownerX",""];
-if !(_owner isEqualTo "" || {getPlayerUID _player isEqualTo _owner}) exitWith {  // crew cannot be moved out if owned by another player.
+if (_owner isNotEqualTo "" && {getPlayerUID _player isNotEqualTo _owner}) exitWith {  // crew cannot be moved out if owned by another player.
```
Comment (Silence-112) [inline A3A/addons/core/functions/Base/fn_moveOutCrew.sqf:45]: doesn't || work just fine here?

You want this exitWith to trigger if:
_owner is not nobody;
OR
_playerUID is not equal to _owner

or am I wrong?

Comment (wersal454) [reply A3A/addons/core/functions/Base/fn_moveOutCrew.sqf:45]: I want it to exit if player is not equal owner AND owner is defined

**Review by Silence-112 — APPROVED** [review]
Seems fine

**Comment by wersal454** [comment]
Tested myself on LAN, alone, works fine. Though would be nice if someone tested on dedicated with other players

**Pull request merged**